src/views/activity/ActivityWindow.vue

```diff
@@ -1,53 +0,0 @@
-<template lang="pug">
-div.mt-3
-  div.row.mb-6
-    div.col-md-6
-      h5 Top Applications
-      aw-summary(:fields="top_apps", :namefunc="e => e.data.app", :colorfunc="e => e.data.app", with_limit)
-    div.col-md-6
-      h5 Top Window Titles
-      aw-summary(:fields="top_titles", :namefunc="e => e.data.title", :colorfunc="e => e.data.app", with_limit)
-
-  div(v-if="periodLength == 'day'")
-    aw-sunburst-clock(:date="date", :afkBucketId="bucket_id_afk", :windowBucketId="bucket_id_window")
```

Comment:
There is now no way to use this visualization.
Personally I never use it, but it's worth mentioning.

Reply:
I added a commented-out version to SelectableVisualization. We'll have to fix it later if there's interest.